packages/governance/src/types.js

```diff
@@ -542,15 +557,15 @@
  */
 
 /**
- * @typedef {Object} GovernedCreatorFacet
+ * @template {object} CF creator facet
+ * @typedef {CF} GovernedCreatorFacet
  * @property {() => ParamManagerRetriever} getParamMgrRetriever - allows accessing
  *   and updating governed parameters. Should only be directly accessible to the
  *   contractGovernor
- * @property {() => T} getLimitedCreatorFacet - the creator
+ * @property {() => CF & LimitedCreatorFacet} getLimitedCreatorFacet - the creator
```

Comment:
`LimitedCreatorFacet` is defined as a generic type with original CF.
```suggestion
 * @property {() => LimitedCreatorFacet<CF>} getLimitedCreatorFacet - the creator
```
The union would make sense if we made the type be "additions to a governed creator facet". We've talked about the problems with that name but it's probably better to do in another PR.

Reply:
done